src/OSPSuite.SimModel/src/ExplicitFormula.cpp

```diff
@@ -449,4 +456,10 @@ void ExplicitFormula::UpdateIndicesOfReferencedVariables()
 		_formula->UpdateIndicesOfReferencedVariables();
 }
 
+void ExplicitFormula::UpdateScaleFactorOfReferencedVariable(const int id, const double ODEScaleFactor)
+{
+	if (_formula != NULL)
+		_formula->UpdateScaleFactorOfReferencedVariable(id, ODEScaleFactor);
+}
```

Comment:
the same: not needed

Reply:
It is, otherwise
https://github.com/Open-Systems-Pharmacology/R-Toolbox/issues/52
does not work.